right-for-life_front/src/components/Article/Article.jsx

```diff
@@ -25,9 +29,16 @@ export const Article = ({article}) => {
         >
           {calcAge(Number(date))} назад
         </aside>
-        <p className="font-medium text-lightgray-600">
+        <p className="font-medium text-lightgray-600 mb-12">
           {text}
         </p>
+        { gallery && gallery.length ? <div className="mb-10">
+          <ArticleImageGallery images={gallery} />
+        </div> : null }
+        { videos && videos.length ? videos.map(video =>
```

Comment:
It would be great to embed videos as slides in the gallery instead of inserting them as separate blocks.
Looks that react-image-gallery supports embed from Youtube.